### ISSUE TITLE
65 - Gamification on home screen once a task is done

### ISSUE THREAD

**Issue by JeremyRolleston** [JeremyRolleston/active8me#65]
Whenever a task is done (i.e. someone clicks done and then goes through the congratulations page, then the sharing page) ... they are ultimately taken back to the home screen.

At this point 2 things happen to the task on the home screen (the one they have just done):
1. An orange tick appears on the "brick" so that the user again gets confirmation that the task is completed.
2. That "brick" disappears. Almost like when you delete a file and it goes to trash can. Like a "whoosh" as it disappears. Maybe even with sound.

**Comment by JeremyRolleston** [comment]
Already done
